scripts/delayed.js

```diff
@@ -1,7 +1,30 @@
 // eslint-disable-next-line import/no-cycle
 import { loadScript, sampleRUM } from './aem.js';
 
+function includeExternalScripts() {
+  const scripts = [
+    'https://icici-securities.allincall.in/files/deploy/embed_chatbot_11.js?version=1.1',
+    'https://bat.bing.com/p/insights/s/0.7.24',
+    'https://bat.bing.com/p/insights/t/20131149',
+    'https://bat.bing.com/bat.js',
+    'https://cdnjs.cloudflare.com/ajax/libs/socket.io/3.1.0/socket.io.min.js',
+    'https://snap.licdn.com/li.lms-analytics/insight.min.js',
+    'https://static.ads-twitter.com/uwt.js',
+    'https://a.quora.com/qevents.js',
+    'https://www.googletagmanager.com/gtag/destination?id=AW-978034268&l=dataLayer&cx=c',
+    'https://www.google-analytics.com/analytics.js',
+    'https://www.googletagmanager.com/gtag/js?id=G-2KB04WCCC2&l=dataLayer&cx=c',
+    'https://www.googletagmanager.com/gtm.js?id=GTM-WF9LTLZ',
```

Comment:
I think some of these get auto-added as a consequence of adding GTM script. So we may not need to add all of them but just the ones that are added by the serverside.

Reply:
yes, we might not need the complete list. IMO GTM adds Quora events since those events are already part of pr https://github.com/aemsites/icicidirect/pull/76. Checking this further. 
This PR in current shape is not ready for merge. I was just checking how inclusion of these works. 
I will put a draft label to this pr

Reply:
@sdmcraft as mentioned here https://github.com/aemsites/icicidirect/pull/76#issuecomment-2021924092
most of it covered via https://github.com/aemsites/icicidirect/pull/80 and https://github.com/aemsites/icicidirect/pull/76.

I will update this to add bing library